src/main/java/forest/detector/service/HtmlParser.java

```diff
@@ -0,0 +1,137 @@
+package forest.detector.service;
+
+import forest.detector.dao.entity.Ticket;
+import forest.detector.dao.entity.Tract;
+import forest.detector.dao.repository.TicketRepository;
+import forest.detector.dao.repository.TractRepository;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+import java.io.IOException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class HtmlParser {
+
+//    public static void main(String[] args) throws IOException, ParseException {
```

Comment:
create unit test instead of using main method
https://github.com/searhiy/java-project-template/blob/master/src/test/java/org/example/service/PetServiceTest.java